src/dss/initialize.py

```diff
@@ -26,14 +31,22 @@ def initialize(lightkube_client: Client) -> None:
         None
     """
     # Path to the manifests YAML file
-    manifests_file = os.path.join(os.path.dirname(__file__), "manifests.yaml")
+    manifests_files = [
+        Path(Path(__file__).parent, MANIFEST_TEMPLATES_LOCATION, "dss_core.yaml.j2"),
+        Path(Path(__file__).parent, MANIFEST_TEMPLATES_LOCATION, "mlflow_deployment.yaml.j2"),
+    ]
+
+    config = {
+        "name": MLFLOW_DEPLOYMENT_NAME,
+        "namespace": DSS_NAMESPACE,
```

Comment:
Will everything be deployed in the same namespace?

Reply:
yeah, I think the spec says everything in dss is in a single namespace.  Unless I misunderstood